src/bot/actions.js

```diff
@@ -95,7 +96,7 @@ const dripLater = async (sender, address, time) => {
   const { amount } = dripActions[DRIP_TYPE.LATER];
   const { data: { hash, providerId } } = res;
 
-  return `I will send ${amount} NEU to address ${address} at ${time} UTC. Extrinsic hash: ${hash}. Your provided_id: ${providerId}.`;
+  return `I will send ${amount} ${tokenSymbol} to address ${address} at ${time} UTC. Extrinsic hash: ${hash}. Your provided_id: ${providerId}.`;
 }
 
 const dripSwag = async (sender, address) => {
```

Comment:
**Remove this hourly drip function completely**. We don’t need it, because
a) it’s not really an automation from user’s wallet
b) it will spam our turing staging chain.

Reply:
Fixed